### ISSUE TITLE
`bundle dump` shows `mas` package not in `mas list`

### ISSUE THREAD

**Issue by lsloan** [Homebrew/homebrew-bundle#1106]
I recently ran `brew bundle dump` and included in the resulting `Brewfile` is this line for Slack:

```txt
mas "Slack", id: 803453959
```

However, when I run `mas list`, it doesn't show Slack:

```txt
937984704   Amphetamine  (5.2.2)
497799835   Xcode        (13.3.1)
405843582   Alfred       (1.2)
```

Where does `bundle dump` get this info about Slack?  In general, where does `bundle dump` get info about any `mas` apps?

**Comment by MikeMcQuaid** [comment]
It just runs `mas list`: https://github.com/Homebrew/homebrew-bundle/blob/48497882863432d8f0bd7dd6727df325a0f00b35/lib/bundle/mac_app_store_dumper.rb#L15

**Comment by lsloan** [comment]
Based on that info, I checked the output of `mas list` again to be sure Slack wasn't included.  It was not.  I ran `brew bundle dump` again, then examined the new `Brewfile`.  It didn't include Slack this time.

I think the problem was that I had used the Mac App Store to install Slack at one time, but I had just switched to using Homebrew to install it from a cask instead.  I had also been applying an update for XCode via the App Store at the time.  So, maybe I had some conflicting operations going on that confused `mas list`.

Thanks for the info.  I'll close this issue.
